controllers/operands/networkAddons.go

```diff
@@ -140,10 +140,13 @@ func NewNetworkAddons(hc *hcov1beta1.HyperConverged, opts ...string) (*networkad
 
 	cnaoSpec := networkaddonsshared.NetworkAddonsConfigSpec{
 		Multus:      &networkaddonsshared.Multus{},
-		LinuxBridge: &networkaddonsshared.LinuxBridge{},
 		KubeMacPool: &networkaddonsshared.KubeMacPool{},
 	}
 
+	if !hcoutil.GetClusterInfo().IsOpenshift() {
+		cnaoSpec.LinuxBridge = &networkaddonsshared.LinuxBridge{}
```

Comment:
We still need to deploy this, even on OCP. Otherwise we never create the needed symlinks downstream, no?

Reply:
Right, will update as we talked offline